Dockerfile.fips

```diff
@@ -0,0 +1,52 @@
+# This version of Go is a Go+BoringCrypto release for FIPS 140-2 compliance
+FROM us-docker.pkg.dev/google.com/api-project-999119582588/go-boringcrypto/golang:1.18.5b7 as build-fips
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download -x
+COPY . .
+RUN rm -rf /src/webconsole/dist
+RUN GOOS=linux GOARCH=amd64 WEBCONSOLE=default make immuadmin-static immudb-fips
+RUN mkdir /empty
+
+### distroless FIPS 140-2
+FROM gcr.io/distroless/base:nonroot AS distroless-fips
+LABEL org.opencontainers.image.authors="Codenotary Inc. <info@codenotary.com>"
+
+ARG IMMU_UID="65532"
+ARG IMMU_GID="65532"
+ARG IMMUDB_HOME="/usr/share/immudb"
+
+WORKDIR /usr/bin
+COPY --from=build-fips /src/immudb .
+COPY --from=build-fips /src/immuadmin .
+COPY --from=build-fips "/etc/ssl/certs/ca-certificates.crt" "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
This should not be necessary, distroless should already contain ca-certificates.crt file  (or a compatible alternative)